app/lib/efile/ny/it201.rb

```diff
@@ -3,12 +3,13 @@ module Ny
     class It201 < ::Efile::TaxCalculator
       attr_reader :lines
 
-      def initialize(year:, filing_status:, intake:, direct_file_data:, dependent_count:, include_source: false)
+      def initialize(year:, filing_status:, intake:, direct_file_data:, eligibility_lived_in_state:, dependent_count:, include_source: false)
         @year = year
 
         @filing_status = filing_status # single, married_filing_jointly, that's all we support for now
         @intake = intake
         @direct_file_data = direct_file_data
+        @eligibility_lived_in_state = eligibility_lived_in_state
```

Comment:
Could this be replaced with `@intake.eligibility_lived_in_state`, since `intake` should be an instance of `StateFileNyIntake`? Could simplify the initialize parameters.

Reply:
yeah that's fine, i'll update it. i was following the pattern that was being used before but it look like it's been changed since i started working on this.

Reply:
actually, filing_status and direct_file_data are also coming from the intake object. not sure what the reason was for them being passed in separately instead of just passing intake before?

Reply:
gonna see about refactoring it but i wonder if this was helpful for setting up tests or something

Reply:
i'm just testing this now, seems like it's gonna work out

Reply:
done and pushed